apps/rush/src/actions/RebuildAction.ts

```diff
@@ -73,7 +73,7 @@ export default class RebuildAction extends CommandLineAction {
     });
     this._vsoParameter = this.defineFlagParameter({
       parameterLongName: '--vso',
-      description: 'Display error messages in the format expected by Visual Studio Online'
+      description: 'Display error messages in the format expected by the Visual Studio Team Services CI log interface'
```

Comment:
I'll change to "build"

---
In reply to: [116345318](https://github.com/Microsoft/web-build-tools/pull/186#discussion_r116345318) [](ancestors = 116345318)